src/mappings/Competition/mapping.ts

```diff
@@ -178,6 +180,7 @@ export function handleNewVote(event: NewVote): void {
     }
     competitionProposal.winningSuggestions = winningSuggestions;
     competitionProposal.totalVotes = competitionProposal.totalVotes.plus(BigInt.fromI32(1));
+    competitionProposal.numWinningSubmissions = BigInt.fromI32(winningSuggestions.length);
     competitionProposal.save();
```

Comment:
we should be consistent here with naming submissions and suggestions

Reply:
But this is the name you asked it to have. So should I change it to `numWinningSuggestions`?

Reply:
I changed it for consistency

Reply:
We are constantly getting confused about "suggestions" and "submissions", hence my error n the request.  Sorry about that.  But subgraph, I think, should itself be consistent.